### DISCUSSION TITLE
Sync System Back to Manager

### DISCUSSION THREAD

**Discussion by ng1991** [uyuni-project/uyuni#8089] [General]
Hi I know this is probably a stupid question but I am new to Uyuni and was wondering how after doing updates I can force a manual check in of my system so the manager picks up the updates that have been applied.

I have looked online but had no joy, we used to use spacewalk but that command is no longer available

**Comment by mcalmer** [comment]
On the Software page there is button to trigger a "software profile update"

**Reply by ng1991** [reply]
sorry for late reply thanks for that
